aucoalesce/coalesce.go

```diff
@@ -454,6 +472,18 @@ func applyNormalization(event *Event) {
 		return
 	}
 
+	event.ECS.Event.Category = norm.ECS.Category.Values
+	event.ECS.Event.Type = norm.ECS.Type.Values
+
+	hasAdditionalNormalization := syscallNorm != nil && syscallNorm != norm
+	if hasAdditionalNormalization {
```

Comment:
this merges normalization information for compound events where they have both an associated syscall and their own normalization

Reply:
I think this one would make for good code comment. These annotation comments are great.